configure.ac

```diff
@@ -439,6 +439,8 @@ if test "x$have_nss" = "xno" -a "x$with_nss" != "xcheck" -a "x$with_nss" != "xno
     AC_MSG_FAILURE([--with-nss was given, but library was not found])
 fi
 
+AC_MSG_NOTICE([NSS is selected, now we really do something])
```

Comment:
someone wrote it;  if you want to remove random cruft, you could send a patch to do a bunch at once?